### DISCUSSION TITLE
TypeError: 'NoneType' object is not subscriptable running the chess example

### DISCUSSION THREAD

**Discussion by techybolek** [microsoft/autogen#335] [Q&A]
When running the chess example https://github.com/microsoft/autogen/blob/main/notebook/agentchat_chess.ipynb
I get an error:

TypeError: 'NoneType' object is not subscriptable 

I've tried both in colab as well as on my local machine.
Full stack:

Player black (to Player white):

Your turn.

--------------------------------------------------------------------------------
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 390 390" width="200" height="200"><desc><pre>r n b q k b n r\np p p p p p p p\n. . . . . . . .\n. . . . . . . .\n. . . . P . . .\n. . . . . . . .\nP P P P . P P P\nR N B Q K B N R</pre></desc><defs><g id="white-pawn" class="white pawn"><path d="M22.5 9c-2.21 0-4 1.79-4 4 0 .89.29 1.71.78 2.38C17.33 16.5 16 18.59 16 21c0 2.03.94 3.84 2.41 5.03-3 1.06-7.41 5.55-7.41 13.47h23c0-7.92-4.41-12.41-7.41-13.47 1.47-1.19 2.41-3 2.41-5.03 0-2.41-1.33-4.5-3.28-5.62.49-.67.78-1.49.78-2.38 0-2.21-1.79-4-4-4z" fill="#fff" stroke="#000" stroke-width="1.5" stroke-linecap="round" /></g><g id="white-knight" class="white knight" fill="none" fill-rule="evenodd" stroke="#000" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"><path d="M 22,10 C 32.5,11 38.5,18 38,39 L 15,39 C 15,30 25,32.5 23,18" style="fill:#ffffff; stroke:#000000;" /><path d="M 24,18 C 24.38,20.91 18.45,25.37 16,27 C 13,29 13.18,
Player white (to Player black):

I'll start with the classic opening, <my_player_name>. e2e4. Let's see how you respond to this.

--------------------------------------------------------------------------------
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-7-bc9baf5cdb2b>](https://localhost:8080/#) in <cell line: 1>()
----> 1 player_black.initiate_chat(player_white, message="Your turn.")

51 frames
[<ipython-input-4-df7b419aae75>](https://localhost:8080/#) in _generate_board_reply(self, messages, sender, config)
     30         # extract a UCI move from player's message
     31         reply = self.generate_reply(self.correct_move_messages[sender] + [message], sender, exclude=[BoardAgent._generate_board_reply])
---> 32         uci_move = reply if isinstance(reply, str) else str(reply["content"])
     33         try:
     34             self.board.push_uci(uci_move)

TypeError: 'NoneType' object is not subscriptable

**Comment by sonichi** [comment]
Have you modified anything in the notebook? It works in my test.

**Reply by techybolek** [reply]
All I did was replace {name} placeholders with names but I'll double check.

**Reply by sonichi** [reply]
oh you don't need to change that. It will be instantiated automatically.
